pkg/gatewayserver/io/udp/incoming.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package udp
+
+import (
+	"sync"
+	"time"
+)
+
+type incoming struct {
+	count   int
```

Comment:
What is the purpose of this var?

Reply:
Used to reset counter. We can drop this if `len(items)` is O(1).

Reply:
It is